src/platform/user/profile/vet360/components/base/VAPEditModalActionButtons.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
```

Comment:
Since I did not want to mess with existing styling/copy in profile 1, I created this new Component based on the existing `Vet360EditModalActionButtons` component. The only differences are that I 

(1) Changed the 'Delete' button to be a link with updated copy
(2) Ensured the buttons are `flex-wrap: wrap`

Reply:
seems fair :)